spec/rspec/rails/configuration_spec.rb

```diff
@@ -252,4 +252,10 @@ def in_inferring_type_from_location_environment
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
   end
+
+  if ::Rails::VERSION::STRING > '5'
+    it "#file_fixture_path defaults to 'spec/fixtures/files'" do
```

Comment:
@morgoth this it statement doesn't scan cleanly.

Can you change it to `it "has a default #file_fixture_path of 'spec/fixtures/files'"`?